tests/test_SklearnDecisionTreeConverters.py

```diff
@@ -5,18 +5,54 @@
 # --------------------------------------------------------------------------
 
 import unittest
+import numpy as np
+from pandas import DataFrame
 from sklearn.tree import DecisionTreeClassifier
 from sklearn.tree import DecisionTreeRegressor
+from sklearn.datasets import make_classification
 from skl2onnx.common.data_types import onnx_built_with_ml
+from skl2onnx.common.data_types import FloatTensorType
+from skl2onnx import convert_sklearn
 from test_utils import (
     dump_one_class_classification,
     dump_binary_classification,
     dump_multiple_classification,
 )
 from test_utils import dump_multiple_regression, dump_single_regression
+from onnxruntime import InferenceSession
 
 
 class TestSklearnDecisionTreeModels(unittest.TestCase):
+    @unittest.skipIf(not onnx_built_with_ml(),
+                     reason="Requires ONNX-ML extension.")
+    def test_decisiontree_classifier1(self):
+        model = DecisionTreeClassifier(max_depth=2)
+        X, y = make_classification(10, n_features=4, random_state=42)
+        X = X[:, :2]
+        model.fit(X, y)
+        initial_types = [('input', FloatTensorType((1, X.shape[1])))]
+        model_onnx = convert_sklearn(model, initial_types=initial_types)
+        sess = InferenceSession(model_onnx.SerializeToString())
+        res = sess.run(None, {'input': X.astype(np.float32)})
+        pred = model.predict_proba(X)
+        if res[1][0][0] != pred[0, 0]:
+            raise AssertionError("{}\n--\n{}".format(pred, DataFrame(res[1])))
```

Comment:
Any reason for not using dump_data_and_model()?

Reply:
For investigation, I need more details results.

Reply:
If this is just for investigation, why are you checking it in?

Reply:
To run it on multiple devices. For some reasons, it works on Linux but not on Windows. I'll probably leave it that way because dump_model also dumps the model on disk and it is not needed for this test. It is faster this way.

Reply:
Okay